### DISCUSSION TITLE
How to set a secondary header for a resource

### DISCUSSION THREAD

**Discussion by dissto** [filamentphp/filament#5281] [Help]
Is it currently possible to set a page "sub" header? Some sort of an introduction text to a resource? 

![Web capture_17-12-2022_215852_127 0 0 1](https://user-images.githubusercontent.com/11778632/208265837-7071f8ba-7a3f-482a-a58d-8ce8c710ad67.jpeg)

**Comment by danharrin** [comment]
I think there's `$subheading` / `getSubheading()` for page classes. So the `ListGroups` class

**Reply by ArtMin96** [reply]
Right.

```php
protected function getSubheading(): string | Htmlable | null
{
    return 'Subheading';
}
```
